src/app/portlets/dot-edit-content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts

```diff
@@ -30,42 +43,72 @@ describe('DotEditPageToolbarComponent', () => {
     });
 
-    it('should have a page title', () => {
-        expect(de.query(By.css('.edit-page-toolbar__page-title'))).toBeTruthy();
-    });
-
-    it('should receive page title as a param', () => {
+    it('should set page title', () => {
         component.pageTitle = 'Hello World';
         const pageTitleEl: HTMLElement = de.query(By.css('.edit-page-toolbar__page-title')).nativeElement;
         fixture.detectChanges();
 
         expect(pageTitleEl.textContent).toEqual('Hello World');
     });
 
+    it('should set page url', () => {
+        component.pageUrl = '/test/test';
+        const pageUrlEl: HTMLElement = de.query(By.css('.edit-page-toolbar__page-url')).nativeElement;
+        fixture.detectChanges();
+
+        expect(pageUrlEl.textContent).toEqual('/test/test');
+    });
+
     it('should have a primary action button', () => {
-        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__primary-action'));
+        fixture.detectChanges();
+        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__save'));
         expect(primaryAction).toBeTruthy();
 
         const primaryActionEl: HTMLElement = primaryAction.nativeElement;
         expect(primaryActionEl.textContent).toEqual('Hello');
     });
 
+    it('should have lock page component', () => {
+        const lockSwitch: DebugElement = de.query(By.css('.edit-page-toolbar__locker'));
+        expect(lockSwitch !== null).toEqual(true);
+    });
+
+    it('should NOT have an action split button', () => {
+        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__actions'));
+        expect(primaryAction).toBeFalsy();
+    });
+
+    it('should have an action split button', () => {
+        component.pageWorkflows = [{ name: 'Workflow 1', id: 'one' }, { name: 'Workflow 2', id: 'two' }];
+        fixture.detectChanges();
+        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__actions'));
+        expect(primaryAction).toBeTruthy();
+        expect(primaryAction.name).toEqual('p-splitButton', 'is a splitbutton');
+    });
+
+    it('should set action split buttons params', () => {
+        component.pageWorkflows = [{ name: 'Workflow 1', id: 'one' }, { name: 'Workflow 2', id: 'two' }];
+        fixture.detectChanges();
+
+        expect(component.actionsButton.label).toEqual('Acciones', 'actions label is set');
+        expect(component.actionsButton.model).toEqual([{ label: 'Workflow 1' }, { label: 'Workflow 2' }]);
+    });
+
     it('should emit save event on primary action button click', () => {
         component.canSave = true;
 
         fixture.detectChanges();
 
-        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__primary-action'));
+        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__save'));
 
         let res;
-        component.save.subscribe(event => {
+        component.save.subscribe((event) => {
```

Comment:
Parenthesis are optional when there's only one parameter, is there any reason why you added it?

Reply:
Probably prettier added it.